tests/unit/IncrementButton.spec.js

```diff
@@ -1,28 +1,57 @@
+
 import {
   render, screen, fireEvent,
 } from '@testing-library/vue';
 import IncrementButton from '@/components/generics/buttons/IncrementButton.vue';
```

Comment:
Btw! 

I wouldn't have called the component `IncrementButton`.

With that name, I was expecting to test a Button. 

But this component is more than that:
1. It is a message.
2. A button.
3. Another message.